jekyll/_cci2/security.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: classic-docs
+title: "Security Features"
+category: [administration]
+order: 5
+description: "CircleCI security features."
+---
+
+This document outlines security features built into CircleCI and related integrations.
+
+* TOC 
+{:toc}
+
+## Overview
+Security is our top priority at CircleCI, we are proactive and we act on security issues immediately. Report security issues to <security@circleci.com> with an encrypted message using our security team's GPG key (ID: 0x4013DDA7, fingerprint: 3CD2 A48F 2071 61C0 B9B7 1AE2 6170 15B8 4013 DDA7).
+
+## Encryption
+CircleCI uses HTTPS or SSH for all networking in and out of our service including from the browser to our services application, from the services application to your builder fleet, from our builder fleet to your source control system, and all other points of communication. In short, none of your code or data travels to or from CircleCI without being encrypted unless you have code in your builds that does so at your discretion. Operators may also choose to go around our SSL configuration or not use TLS for communicating with underlying systems.
+
+The nature of CircleCI is that our software has access to your code and whatever data that code interacts with. All builds on CircleCI run in sandbox (specifically, an LXC container) that stands alone from all other builds and is not accessible from the Internet or from your own network. The build container pulls code via git over SSH. Your particular test suite or build configuration may call out to external services or integration points within your network, and the response from such calls will be pulled into your build and used by your code at your discretion. After a build is complete the container that ran the build is destroyed and rebuilt. All environment variables you store inside CircleCI are encrypted at rest and sent to your build containers using SSH.
```

Comment:
Change LXC here to say "a Docker container or on-demand VM" and add an "a" in front of "sandbox"

Reply:
thanks, fixed